src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -149,14 +147,17 @@ const AdvancedSettings = ({ intl, courseId }) => {
                     <ul className="setting-items-list p-0">
                       {Object.keys(advancedSettingsData).sort().map((settingName) => {
                         const settingData = advancedSettingsData[settingName];
+                        const editedValue = editedSettings[settingName] !== undefined
```

Comment:
Can you try this approach?
```suggestion
const editedValue = editedSettings[settingName] ?? settingData.value;
```